index.rst

```diff
@@ -59,10 +59,3 @@ How to contribute to this documentation
    :maxdepth: 2
 
    contributing/index
-
-Indices and tables
```

Comment:
Yeah, I had left these in because I didn't know if they would potentially be useful. I haven't gotten to the point of educating myself enough to know if we could make use of any of these. Thoughts?

Reply:
I'm not really sure what to do with them either. I'd be fine cutting them out for now.